cmd/eval.go

```diff
@@ -210,6 +210,8 @@ Set the output format with the --format flag.
     --format=values    : output line separated JSON arrays containing expression values
     --format=bindings  : output line separated JSON objects containing variable bindings
     --format=pretty    : output query results in a human-readable format
+    --format=source    : output partial evaluation results in a source file friendly format
```

Comment:
🤔 hmm how about this? It's a bit tautological, maybe...
```suggestion
    --format=source    : output partial evaluation results in a source format
```

Reply:
Changed 👍